CONTRIBUTING.md

```diff
@@ -80,6 +82,8 @@ yarn
 yarn docs:dev
 ```
 You can now access the documentation site [locally](http://localhost:3000).
+Changes to the docs will hot reload the site. If you make changes to TypeScript files
```

Comment:
`Changes to the docs will hot reload the site.`

This is confusing. Yes, it does hot reload the side. Or, I'm missing something?

Reply:
@oliviertassinari  `yarn docs:dev` has hot module replacement. That's what's writting here. Is there some contradiction or why is this confusing?